### ISSUE TITLE
Can no more change read/unread, star/unstar status in OpenPaaS ESN Inbox emails using James

### ISSUE THREAD

**Issue by StephaneColson** [linagora/james-project#554]
With an email displayed in OpenPaaS/Inbox, each time I read, or change manually the Star/Unstar or Read/Unread status, then the status is not changed in the email and this error occurs in James logs:

```
esn_james       | 15:50:08.351 [WARN ] o.e.j.s.ServletHandler - /jmap
esn_james       | org.apache.james.backends.cassandra.utils.LightweightTransactionException: Can not execute Optional Supplier. 1000 retries.
esn_james       |     at org.apache.james.backends.cassandra.utils.FunctionRunnerWithRetry.lambda$executeAndRetrieveObject$3(FunctionRunnerWithRetry.java:54)
esn_james       |     at java.util.Optional.orElseThrow(Optional.java:290)
esn_james       |     at org.apache.james.backends.cassandra.utils.FunctionRunnerWithRetry.executeAndRetrieveObject(FunctionRunnerWithRetry.java:54)
esn_james       |     at org.apache.james.mailbox.cassandra.mail.CassandraMessageIdMapper.flagsUpdateWithRetry(CassandraMessageIdMapper.java:221)
esn_james       |     ... 88 common frames omitted
esn_james       | Wrapped by: java.lang.RuntimeException: org.apache.james.backends.cassandra.utils.LightweightTransactionException: Can not execute Optional Supplier. 1000 retries.
esn_james       |     at com.google.common.base.Throwables.propagate(Throwables.java:160)
esn_james       |     at org.apache.james.mailbox.cassandra.mail.CassandraMessageIdMapper.flagsUpdateWithRetry(CassandraMessageIdMapper.java:228)
esn_james       |     at org.apache.james.mailbox.cassandra.mail.CassandraMessageIdMapper.lambda$setFlags$14(CassandraMessageIdMapper.java:212)
esn_james       |     at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
esn_james       |     at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184)
esn_james       |     at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
esn_james       |     at java.util.Iterator.forEachRemaining(Iterator.java:116)
esn_james       |     at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
esn_james       |     at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
esn_james       |     at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
esn_james       |     at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151)
esn_james       |     at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174)
esn_james       |     at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
esn_james       |     at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:418)
esn_james       |     at java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:270)
esn_james       |     at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
esn_james       |     at java.util.Iterator.forEachRemaining(Iterator.java:116)
```

**Comment by StephaneColson** [comment]
Same error when trying to download an attachment and this window is diplayed

![browserwarningwhendowloading](https://cloud.githubusercontent.com/assets/1481595/21855231/fc3b379e-d81d-11e6-9ad6-7fea0b624acd.png)

The dowload fail first time, then is ok the second.

**Comment by rouazana** [comment]
Fixed by MAILBOX-285, see #556

**Comment by StephaneColson** [comment]
OK, thanks
